src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/ValidateFeeCallbackHandler.java

```diff
@@ -47,7 +49,9 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse validateFee(CallbackParams callbackParams) {
         var caseData = callbackParams.getCaseData();
         List<String> errors = new ArrayList<>();
-        var fee = feesService.getFeeDataByClaimValue(caseData.getClaimValue());
+        var fee = feesService.getFeeDataByClaimValue(caseData.getClaimValue(),
```

Comment:
@asthamalviya what relevance does Spec Category have on the Fee? Are these changes not for Spec and Unspec?